.github/workflows/deploy-pipeline-aws.yml

```diff
@@ -137,7 +137,7 @@ jobs:
             datateer-docker-pipeline-
 
       - name: Deploy pipeline
-        run: datateer pipeline deploy ${{ env.PIPELINE_NAME }} --environment ${{ env.DATATEER_ENV }}
+        run: datateer pipeline deploy ${{ env.PIPELINE_NAME }} --environment ${{ env.DATATEER_ENV }} --cloud aws --region ${{ env.REGION }} --account ${{ env.ACCOUNT }}
```

Comment:
This is the important change right here. Refactor it to use the variables with the aws prefix and all should be good